README.md

```diff
@@ -20,15 +27,15 @@ across the federation is solely the model parameters and their updates.
 Open Federated Learning (OpenFL) is a Python 3 project developed by Intel Labs and 
 Intel Internet of Things Group. 
 
-![Federated Learning](docs/images/diagram_fl.png)
+![Federated Learning](https://raw.githubusercontent.com/intel/openfl/master/docs/images/diagram_fl.png)
```

Comment:
Shouldn't this just point to `docs/images/diagram_fl.png`?

Reply:
@tonyreina this was done for correct PyPI package page rendering.